src/rpc/rawtransaction.cpp

```diff
@@ -2808,13 +2808,10 @@ struct RawIssuanceDetails
 
 // Appends a single issuance to the first input that doesn't have one, and includes
 // a single output per asset type in shuffled positions.
-void issueasset_base(CMutableTransaction& mtx, RawIssuanceDetails& issuance_details, const CAmount asset_amount, const CAmount token_amount, const std::string& asset_address_str, const std::string& token_address_str, const bool blind_issuance, const uint256& contract_hash)
+void issueasset_base(CMutableTransaction& mtx, RawIssuanceDetails& issuance_details, const CAmount asset_amount, const CAmount token_amount, const CTxDestination& asset_dest, const CTxDestination& token_dest, const bool blind_issuance, const uint256& contract_hash)
 {
-
-    CTxDestination asset_address(DecodeDestination(asset_address_str));
-    CTxDestination token_address(DecodeDestination(token_address_str));
-    CScript asset_destination = GetScriptForDestination(asset_address);
-    CScript token_destination = GetScriptForDestination(token_address);
+    CScript asset_script = GetScriptForDestination(asset_dest);
```

Comment:
You don't need to change this now if you don't feel like it, but in the future it makes review easier not to BOTH rename AND move code in the same commit.

Reply:
Yeah I could have cut the change into more than one commit. I'd be happy to do that. But I see you ack'ed it.